### PR TITLE
Update Icons

### PR DESCRIPTION
**Pull request by LFBarreto**
Updated icons and icons story
localhost_6006__path=_story_asorted-icons--list (1)
﻿

### CODE REVIEW

File: packages/react/src/components/form/BaseInput/index.tsx [hunk 1]
```diff
@@ -125,14 +125,19 @@ export default function Input(props: InputProps): JSX.Element {
   } = props;
   const [focus, setFocus] = React.useState(false);
 
+  const handleChange = useCallback(
```
Comment (jelbaz-ledger) [inline packages/react/src/components/form/BaseInput/index.tsx:128]: ⚠️ This is a breaking change, we need to ensure that the work done in LLD so far gets updated if necessary.

File: packages/react/src/components/form/BaseInput/index.tsx [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-export type CommonProps = InputHTMLAttributes<HTMLInputElement> &
+export type CommonProps = Omit<InputHTMLAttributes<HTMLInputElement>, "onChange"> &
   TypographyProps & {
     disabled?: boolean;
     error?: string;
   };
 
 export type InputProps = CommonProps & {
-  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  onChange: (e: string) => void;
```
Comment (jelbaz-ledger) [inline packages/react/src/components/form/BaseInput/index.tsx:15]: ⚠️ This is a breaking change, so we'll need to ensure that the work done in LLD so far gets updated if necessary.

File: packages/react/src/components/asorted/Icon/Icons.stories.tsx [hunk 3]
```diff
@@ -44,15 +61,32 @@ export default Story;
...
+    <Container>
+      <SearchInput value={search} onChange={setSearch} />
+      <ScrollArea
+        gridTemplateColumns="repeat(auto-fill, minmax(100px, 1fr));"
+        gridTemplateRows="repeat(150px);"
+        gridGap={4}
+        mt={4}
+      >
+        {iconNames
+          .sort((a: string, b: string) => {
```
Comment (jelbaz-ledger) [inline packages/react/src/components/asorted/Icon/Icons.stories.tsx:77]: ✨ A cool thing to implement would be to highlight the icons that match the search string, as I think it's not super obvious that the search bar does not actually filter the icons.

File: packages/react/src/components/asorted/Icon/Icon.tsx [hunk 4]
```diff
@@ -22,7 +22,7 @@ const Icon = ({
   name,
   size = 16,
   color = "currentColor",
-  weight = "Regular",
+  weight = "Medium",
```
Comment (jelbaz-ledger) [inline packages/react/src/components/asorted/Icon/Icon.tsx:25]: This is also breaking, as we might have implemented icons without the prop thinking that Regular would be the default.

**Review by jelbaz-ledger — APPROVED** [review]
👍 LGTM as long as we check that it does not break stuff!

File: packages/react/src/components/form/BaseInput/index.tsx (hunk 2 above)
Comment (LFBarreto) [reply packages/react/src/components/form/BaseInput/index.tsx:15]: i dont think we used or will use any input just yet so it should be fine for now :+1:

File: packages/react/src/components/asorted/Icon/Icons.stories.tsx (hunk 3 above)
Comment (LFBarreto) [reply packages/react/src/components/asorted/Icon/Icons.stories.tsx:93]: just did that :+1:

**Pull request merged**